layers/meta-balena-jetson/conf/layer.conf

```diff
@@ -19,6 +19,6 @@ BBMASK += "/meta-tegra/recipes-graphics/ "
 BBMASK += "/meta-tegra/external/openembedded-layer/recipes-support/opencv/ "
 
 LAYERSERIES_COMPAT_balena-jetson = "warrior"
-KERNEL_ARGS_jetson-xavier = "console=ttyTHS0,115200 OS=l4t fbcon=map:0"
+KERNEL_ARGS_jetson-xavier = "console=ttyTHS0,115200n8 OS=l4t fbcon=map:0 console=tty1 debug loglevel=7"
```

Comment:
Do we want the bootlog on tty1 in production images?

Reply:
No, we dont want that. Here we just override the machine.conf defined cmdline. Specifying console=null at the end for dev builds, like we do here: https://github.com/balena-os/balena-jetson/blob/e4fabce727ae9bb88d3cf4a27b7732477bf7fb61/layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra194-flash-dry_32.2.0.bb#L39 used to disable all the previous defined ones, but I'll check once again to be 100% sure nothing has changed.